### ISSUE TITLE
Track method documentation changes

### ISSUE THREAD

**Issue by rtholmes** [ataraxie/codeshovel#64]
Methods often have documentation that directly precedes their signatures (e.g., javadoc). While a change to documentation is not a change to the method, it is definitely _associated_ with the method. This feels somewhat analogous to the Annotation change support introduced in #1 and should be handled similarly.

**Comment by rtholmes** [comment]
@ishtiaque05 is this something you'd be able to add like you did for #1?

**Comment by braxtonhall** [comment]
Should be easy!
`MethodDeclaration` has `method.hasJavadoc()` and `Javadoc` has `javadoc.toText()`

**Comment by ishtiaque05** [comment]
Yup I can add that up unless @braxtonhall have already implemented it 😄

**Comment by ishtiaque05** [comment]
Java implementation for Ydocchange change: #65 
Python and typescript implementation for Ydocchange: #69
